src/httpserver/http_response.hpp

```diff
@@ -183,14 +144,13 @@ class http_response
         virtual int enqueue_response(MHD_Connection* connection, MHD_Response* response);
 
     protected:
-        std::string content;
-        int response_code;
+        int response_code = -1;
 
         std::map<std::string, std::string, http::header_comparator> headers;
         std::map<std::string, std::string, http::header_comparator> footers;
         std::map<std::string, std::string, http::header_comparator> cookies;
 
-    	friend std::ostream &operator<< (std::ostream &os, const http_response &r);
+      	friend std::ostream &operator<< (std::ostream &os, const http_response &r);
```

Comment:
the indentation must have accidentally changed here. Can we fix it?

Reply:
Reverted.